.github/workflows/release.yml

```diff
@@ -273,8 +273,23 @@ jobs:
       - name: Pack binaries (Unix)
         if: matrix.config.os != 'windows-latest'
         run: |
+          set -e
+
           archive=jormungandr-${{ needs.create_release.outputs.version }}-${{ matrix.config.target }}-${{ matrix.target_cpu }}.tar.gz
+
+          sha256sum ./target/${{ matrix.config.target }}/release/jormungandr | awk '{print $1}' > ./jormungandr.sha256
```

Comment:
Unfortunately, sha256sum is not available out of the box on macOS. Either [install coreutils with brew](https://unix.stackexchange.com/a/484429/415360) or do it with `shasum -a 256` specifically on macOS.

Reply:
Conditionalizing to do these sanity checks only on macOS is simplest and the insanity is only lurking there.

Reply:
Weird enough, but I also have the `shasum` command on my machine. It is a Perl script,  trying to understand where is it coming from.